atlasdb-client/src/main/java/com/palantir/atlasdb/keyvalue/impl/TracingPrefsConfig.java

```diff
@@ -61,6 +62,8 @@ public void run() {
                             tracingPrefConfig.getProperty("min_duration_to_log_ms", "0"));
                     String tableString = tracingPrefConfig.getProperty("tables_to_trace", "");
                     tracedTables = ImmutableSet.copyOf(Splitter.on(",").trimResults().split(tableString));
+                    retryOnDifferentHostOnThriftTimedOutExceptions = Boolean.parseBoolean(
+                            tracingPrefConfig.getProperty("retryOnDifferentHostOnThriftTimedOutExceptions", "false"));
```

Comment:
You are totally correct.  We wanted a small code change that would allow live reloading due to the length of the outage window we were going to use to test this out.  IE. if this caused additional perf problems we wanted to be able to cut it off without an outage.